dll/cpl/sysdm/general.c

```diff
@@ -158,8 +158,8 @@ LRESULT CALLBACK RosImageProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam
                     hCreditsDC = CreateCompatibleDC(hDC);
                     hLogoDC = CreateCompatibleDC(hCreditsDC);
 
-                    if (hCreditsDC == NULL || hLogoDC == NULL)
-                        break;
+                    if (hCreditsDC == NULL || hLogoDC == NULL || hDC == NULL)
```

Comment:
Here seems too late for `hDC == NULL`, which was already used, isn't it?

Reply:
Yes correct. I fix.